cmd/broker/filter/main.go

```diff
@@ -77,6 +118,18 @@ func main() {
 		logger.Fatal("Manager.Start() returned an error", zap.Error(err))
 	}
 	logger.Info("Exiting...")
+
+	// TODO Gracefully shutdown the filter server. CloudEvents SDK doesn't seem
```

Comment:
What do you need?

Reply:
I need to be able to call the Shutdown method of the HTTP transport. Looks like this is actually supported now in the SDK when creating the transport separately, so that's probably what we should do here.

Reply:
you pass in a context that is make with `ContextWithCancel`  to StartReceiver and then use the cancel function.

Reply:
Looks like that already happens in https://github.com/knative/eventing/blob/f7ab5967103139f0b3c6497e7c3dbeded07ed188/pkg/broker/receiver.go#L112, so I'll remove this comment. Thanks for pointing that out @n3wscott!